src/main/java/com/minecolonies/coremod/commands/BackupCommand.java

```diff
@@ -35,11 +37,8 @@ public BackupCommand(@NotNull final String... parents)
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
     {
-        if (!sender.canCommandSenderUseCommand(PERMNUM, "minecolonies backup"))
-        {
-            sender.addChatMessage(new TextComponentString(NO_PERMISSION_MESSAGE));
```

Comment:
I think we still need the "NO_PERMISSION_MESSAGE" feedback if the player isn't opped.